examples/trainer_keras.py

```diff
@@ -28,83 +33,124 @@ def ensure_results_dir():
 
 def generate_data(backcast_length, forecast_length):
     def gen(num_samples):
-        return next(dummy_data_generator(backcast_length, forecast_length,
+        return next(dummy_data_generator_multivariate(backcast_length, forecast_length,
                                          signal_type='seasonality', random=True,
                                          batch_size=num_samples))
 
     x_train, y_train = gen(6_000)
     x_test, y_test = gen(1_000)
-    return x_train, y_train, x_test, y_test
+    return x_train.reshape((x_train.shape[0], x_train.shape[1], 1)), None, y_train.reshape((y_train.shape[0], y_train.shape[1], 1)), x_test.reshape((x_test.shape[0], x_test.shape[1], 1)), None, y_test.reshape((y_test.shape[0], y_test.shape[1], 1))
```

Comment:
Maybe wrap x_train.shape[0] in a variable or define a tiny function to reshape like this.

Reply:
Updated with a tiny function reshape_array (nb_samples, nb_timestamps) -> (nb_samples, nb_timestamps, 1)